packages/main/src/Menu.ts

```diff
@@ -542,9 +546,16 @@ class Menu extends UI5Element {
 		this._parentItemsStack.push(item);
 	}
 
+	_onfocusin(e: CustomEvent<any>): void {
+		const target = e.target as HTMLElement;
+		const menuListItem = target instanceof MenuListItem ? target : (target.getRootNode() as any).host;
+		const menuItem = menuListItem.associatedItem as MenuItem;
+		const mainMenu = this._findMainMenu(menuItem);
+		mainMenu?.fireEvent("ui5-item-focusin", { opener: menuListItem, menuItem });
```

Comment:
Regarding the event parameters:
 - `opener` is too tailored for the use case where you want to open a Popover from there. I would call it `ref` to be aligned with other components
  - `menuItem` should be just `item`, the word "menu" is redundant as the context is clear.
  
  Take for example this code from an existing comopnent:
  
  ```ts
  suggestion && suggestion.fireEvent("mouseout", {
			item: suggestion,
			targetRef: item,
		});
		```
		
		It seems to be the same situation: `item` is the logical item (here it would be the menu item) and `ref` is the physical item (here the list item). I would have the same API here for this event.

Reply:
done